bot/exts/info/patreon.py

```diff
@@ -0,0 +1,162 @@
+
+        await sending_channel.send(
+            embed=discord.Embed(
+                description=message,
+                colour=colour
+            )
+        )
+
+    async def send_current_supporters(self, channel: discord.TextChannel) -> None:
+        """Send the current list of patreon supporters, sorted by tier level."""
+        await self.bot.wait_until_guild_available()
+
+        guild = self.bot.get_guild(constants.Guild.id)
+
+        tier_1_or_above_patrons = set(guild.get_role(constants.Roles.patreon_tier_1).members)
+        tier_2_or_above_patrons = set(guild.get_role(constants.Roles.patreon_tier_2).members)
+        tier_3_patrons = set(guild.get_role(constants.Roles.patreon_tier_3).members)
+
+        tier_1_patrons = tier_1_or_above_patrons - tier_2_or_above_patrons - tier_3_patrons
+        tier_2_patrons = tier_2_or_above_patrons - tier_3_patrons
+
+        tier_1_patrons = {f"{patron.mention} ({patron.name}#{patron.discriminator})" for patron in tier_1_patrons}
+        tier_2_patrons = {f"{patron.mention} ({patron.name}#{patron.discriminator})" for patron in tier_2_patrons}
+        tier_3_patrons = {f"{patron.mention} ({patron.name}#{patron.discriminator})" for patron in tier_3_patrons}
+
+        embed_list = []
+
+        embed_patron_info = discord.Embed(
+            title="Patreon Supporters",
+            description=(
+                "We use the money from Patreon to offer excellent prizes for all of our events. Prizes like t-shirts, "
+                "stickers, microcontrollers that support CircuitPython, or maybe even a mechanical keyboard.\n\n"
+
+                "You can read more about how Patreon supports us, or even support us yourself, on our Patreon page "
+                "[here](https://pydis.com/patreon)!"
+            )
+        )
+
+        embed_list.append(embed_patron_info)
+
+        if tier_1_patrons:
+            embed_tier_1 = discord.Embed(
+                title="Tier 1 patrons",
+                description="\n".join(tier_1_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_1).colour
+            )
+            embed_list.append(embed_tier_1)
+
+        if tier_2_patrons:
+            embed_tier_2 = discord.Embed(
+                title="Tier 2 patrons",
+                description="\n".join(tier_2_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_2).colour
+            )
+            embed_list.append(embed_tier_2)
+
+        if tier_3_patrons:
+            embed_tier_3 = discord.Embed(
+                title="Tier 3 patrons",
+                description="\n".join(tier_3_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_3).colour
+            )
+            embed_list.append(embed_tier_3)
+
+        await channel.send(embeds=embed_list)
+
+    @commands.command("patrons")
+    async def current_supporters_command(self, ctx: commands.Context) -> None:
```

Comment:
this command name of `patrons` vs `patreon` for the below command, will likely be a source of confusion. A different way to go about this may be making the below informational command a command *group*, and making the list a subcommand. 

The docs for doing so are located here: https://discordpy.readthedocs.io/en/master/ext/commands/api.html#group

Reply:
I agree it is a bit confusing, i'll think about how best to do this

Reply:
Done